data/config-schema/src/main/resources/net/corda/schema/configuration/messaging/1.0/corda.messaging.json

```diff
@@ -55,7 +55,10 @@
                     "properties": {
                       "maxPollRecords": {
                         "description": "Max records to poll from db in a single poll.",
-                        "type": "integer"
+                        "type": "integer",
```

Comment:
@fowlerrr what is the current default that we use? I'm a bit surprised we weren't specifying one here before actually, does that imply that we're ignoring this entirely?

Reply:
This is in one of the default configs.
```
# Default properties for user configurable fields
consumer = {
    maxPollRecords = 100
}
```
It gets merged in with the config passed on subscription creation, which I _think_ comes from the config read service.

Reply:
I have put defaulr value to 100.